backend/api/views/annotation_campaign.py

```diff
@@ -197,6 +219,10 @@ def report(self, request, pk=None):
                         + timedelta(seconds=(result.end_time or max_time))
                     ).isoformat(timespec="milliseconds"),
                     "1" if is_box else "0",
+                    result.confidence_indicator.label
+                    if result.confidence_indicator is not None
+                    else "",
```

Comment:
Wow is this a multi-line ternary expression ? Kind of a meta concept :grin:, cool but not very readable : if it's too big for one line maybe you can set it in a variable above ?

Reply:
You added another multi-line ternary expression in your fix commit :sweat_smile: